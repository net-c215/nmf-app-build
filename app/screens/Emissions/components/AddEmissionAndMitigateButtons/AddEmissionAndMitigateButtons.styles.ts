import { StyleSheet } from "react-native";

import { Layout } from "constant";

const button = {
  flex: 1,
  marginBottom: 20,
};
export default StyleSheet.create({
  buttonView: {
    flexDirection: "row",
    position: "absolute",
    bottom: 0,
    right: 0,
    left: 0,
  },
  buttonLeft: {
    ...button,
    marginLeft: Layout.PADDING_HORIZONTAL,
    marginRight: Layout.PADDING_HORIZONTAL / 2,
  },
  buttonRight: {
    ...button,
    marginRight: Layout.PADDING_HORIZONTAL,
    marginLeft: Layout.PADDING_HORIZONTAL / 2,
  },
});
