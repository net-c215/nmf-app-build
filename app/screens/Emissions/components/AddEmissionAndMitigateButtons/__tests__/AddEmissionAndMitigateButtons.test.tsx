import React from "react";
import renderer from "react-test-renderer";
import AddEmissionAndMitigateButtons from "../AddEmissionAndMitigateButtons";

jest.unmock("../AddEmissionAndMitigateButtons");

describe("AddEmissionAndMitigateButtons", () => {
  describe("rendering", () => {
    it("renders correctly", () => {
      const tree = renderer.create(<AddEmissionAndMitigateButtons />).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
