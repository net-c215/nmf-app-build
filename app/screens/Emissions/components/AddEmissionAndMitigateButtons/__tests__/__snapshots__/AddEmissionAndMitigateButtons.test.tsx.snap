// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddEmissionAndMitigateButtons renders correctly 1`] = `
<View
  style={
    Object {
      "bottom": 0,
      "flexDirection": "row",
      "left": 0,
      "position": "absolute",
      "right": 0,
    }
  }
>
  <Button.Primary
    black={true}
    onPress={[Function]}
    style={
      Object {
        "flex": 1,
        "marginBottom": 20,
        "marginLeft": 14,
        "marginRight": 7,
      }
    }
    textType="Primary"
  >
    <Text.Primary
      bold={true}
      center={true}
      white={true}
    >
      Mitigate
    </Text.Primary>
  </Button.Primary>
  <Button.Primary
    onPress={[Function]}
    style={
      Object {
        "flex": 1,
        "marginBottom": 20,
        "marginLeft": 7,
        "marginRight": 14,
      }
    }
    textType="Primary"
  >
    <Text.Primary
      bold={true}
      center={true}
      white={true}
    >
      Add
    </Text.Primary>
  </Button.Primary>
</View>
`;
